### ISSUE TITLE
Induced subgraph occurrence test labels

### ISSUE THREAD

**Issue by zhengziyu77** [Cqkkkkkk/DAGPrompT#3]
`
data process

          need_node_num = smallest_size - len(subset)
          pos_nodes = torch.argwhere(data.y == int(current_label))
          candidate_nodes = torch.from_numpy(
              np.setdiff1d(pos_nodes.numpy(), subset.numpy()))
          candidate_nodes = candidate_nodes[torch.randperm(
              candidate_nodes.shape[0])][0:need_node_num]
          subset = torch.cat(
              [torch.flatten(subset), torch.flatten(candidate_nodes)])
`
The second line **pos_nodes = torch.argwhere(data.y == int(current_label))** implicitly uses the labeling information to construct a homophilic induced subgraph.This resulted in biased actual results, and removal of this component resulted in a significant decrease in model performance.

**Comment by Cqkkkkkk** [comment]
Thank you for your comment. The data preprocessing code is inherited from established prior work, and in practice, the conditional statement

```python
if len(subset) < smallest_size:
```

is never triggered in my recent trials, as `len(subset)` consistently exceeds `smallest_size`. Regarding the drop in model performance, this can be attributed to several factors, such as differences in CUDA environment versions or the inherent non-determinism of certain computation functions, both of which can introduce performance variability from subtle code or hardware changes. For optimal results, we recommend either using the pre-processed dataset directly or performing a grid search over hyperparameters, as the default values provided may not be universally optimal.

**Comment by zhengziyu77** [comment]
There are many isolated nodes in the heterophilic graph, which will trigger the function.

**Comment by Cqkkkkkk** [comment]
Thank you for pointing this out. Upon closer inspection, I realized I was monitoring the wrong function—the code snippet in question is indeed triggered multiple times. This does introduce a form of data leakage. I appreciate the careful observation!

**Comment by zhengziyu77** [comment]
Can you update the results in the paper after modifying the code which will contribute to a proper comparison.

**Comment by Cqkkkkkk** [comment]
Currently all experiments use an identical preprocessing pipeline—borrowed from established works such as GPF, GraphPrompt, AllInOne, and GCOPE—so the comparison is internally fair. If time allows, we will rerun the experiments under a label‑agnostic  setting and publish the updated results on GitHub or include them as an addendum to the paper. Thank you for your suggestion.
